force-app/main/default/classes/TestApexBasicAssignment.cls

```diff
@@ -0,0 +1,34 @@
+@isTest
+public class TestApexBasicAssignment {
+    @isTest static void testGetStudentList(){
+        Class__c cls1 = new Class__c(Name='PKG',Board__c='RBSE',Fee__c=15000,Custom_Status__c='New',Max_Size__c=5);
+        insert cls1;
+        Student__c std1 = new Student__c(Name='Aish',First_Name__c='Aish',Last_Name__c='warya',Date_Of_Birth__c=Date.newInstance(1997, 07, 22),Sex__c='FEMALE',Email__c='sfd@cscd.vfd',Class__c=cls1.Id);
+        Student__c std2 = new Student__c(Name='Arindam',First_Name__c='Arindam',Last_Name__c='ssf',Date_Of_Birth__c=Date.newInstance(1997, 06, 03),Sex__c='FEMALE',Email__c='sfd@cscd.vfd',Class__c=cls1.Id);
+        insert std1;
+        insert std2;
+        List<Student__c> stdList = ApexBasicAssignment.getStudentsList(cls1.Name);
+        System.assertEquals(2, stdList.size());
+    }
+    
+    @isTest static void testmapClassAndStudents(){
+        Class__c cls1 = new Class__c(Name='PKG',Board__c='RBSE',Fee__c=15000,Custom_Status__c='New',Max_Size__c=5);
+        insert cls1;
+        Student__c std1 = new Student__c(Name='Aish',First_Name__c='Aish',Last_Name__c='warya',Date_Of_Birth__c=Date.newInstance(1997, 07, 22),Sex__c='FEMALE',Email__c='sfd@cscd.vfd',Class__c=cls1.Id);
+        Student__c std2 = new Student__c(Name='Arindam',First_Name__c='Arindam',Last_Name__c='ssf',Date_Of_Birth__c=Date.newInstance(1997, 06, 03),Sex__c='FEMALE',Email__c='sfd@cscd.vfd',Class__c=cls1.Id);
+        insert std1;
+        insert std2;
+        Map<String,List<Student__c>> listOfStudent = ApexBasicAssignment.mapClassAndStudents();
+        System.assertEquals(1,listOfStudent.size());
+    }
+    
+    @isTest static void testGetAllObjects(){
+        List<String> sObjectList = ApexBasicAssignment.GetAllsObject();
+        System.assertEquals(614,sObjectList.size());
```

Comment:
Try to check for not null and not empty for this type of thing instead of checking the exact number because if you added any new custom object then the count would be increased and your test is going to fail, which is not ideal.